frontend/pages/Parts/index.tsx

```diff
@@ -28,6 +28,15 @@ export const getServerSideProps: GetServerSideProps<AppPageProps> = async (
       'public, s-maxage=600, stale-while-revalidate=1200'
    );
 
+   const isEN = !!context.req.headers.host?.match(/www\.ifixit\.com/g);
+
+   if (!isEN) {
```

Comment:
Shouldn't we be redirecting if this is the case? How does condition get executed?